experimental/framework/fluid-static/src/types.ts

```diff
@@ -74,10 +74,13 @@ export interface ContainerSchema {
 }
 
 /**
- * Event that triggers when the roster of members in the Fluid session change
+ * Events that trigger when the roster of members in the Fluid session change.
+ * Only changes that would be reflected in the returned map of IServiceAudience's getMembers method
+ * will emit events.
  */
 export interface IServiceAudienceEvents<M extends IMember> extends IEvent {
-    (event: "membersChanged", listener: (members: Map<string, M>) => void): void;
+    (event: "membersChanged", listener: () => void): void;
+    (event: "addMember" | "removeMember", listener: (clientId: string, member: M) => void): void;
```

Comment:
nit: Would we want the events to perhaps be named "memberAdded" and "memberRemoved" to keep it consistent in naming style and tense with other events?

Reply:
Yeah I think that's better.  I kept the same names as with IAudience but I think those are probably poor even disregarding consistency.